frontend/packages/dev-console/integration-tests/features/getting-started/getting_started _to_dev_perspective.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Create Sample Application
+    As a user I want to create the Sample Application from Add options
+
+Background:
+    Given user is developer
+
+@regression
+Scenario: Logging to openshift devconsole
+   Given user has devloper credentials
+   When user logins as a developer
+   Then user is taken to developer perspective directly 
```

Comment:
If user before logging out  was in the developer perspective, then it will be taken to developer perspective. Can you please check whether they have added such feature. Also please add the new line.

Reply:
or we can start with user logging out from kubeadmin and then logging back in as a developer. WDYT about this?

Reply:
If you update to the above scenario, then you could change the background as well.

Reply:
When user login as a developer it  will be taken to developer perspective which is part of the epic

Reply:
It is a new feature where when developer login( for the first time also) it will be taken to developer perspective directly, you can check the epic